docs/upgrade-percona-repos.md

```diff
@@ -0,0 +1,111 @@
+# Upgrade using the Percona repositories
+
+We recommend using the Percona repositories to upgrade your server.
+
+Find the instructions on how to enable the repositories in the following documents:
+
+* [Percona APT Repository](apt-repo.md)
+
+* [Percona RPM Repository](yum-repo.md)
+
+
+=== "DEB-based distributions"
+
+    Run the following commands as root or use the `sudo` command.
+
+
+    1. Make a full backup (or dump if possible) of your database. Move the database configuration file, `my.cnf`, to another directory to save it. If the configuration file is not moved, it can be overwritten.
+
+
+    2. Stop the server with the appropriate command for your system:
+   
+       ```{.bash data-prompt="$"}
+       systemctl stop mysql`
+       ```
+
+    3. Modify the database configuration file, `my.cnf`, as needed.
+
+    4. Install Percona Server for MySQL:
+
+        ```{.bash data-prompt="$"}
+        $ sudo apt update
+        $ sudo apt install curl
+        $ curl -0 https://repo.percona.com/apt/percona-release_latest.generic_all.deb 
+        $ sudo apt install gnupg2 lsb-release ./percona-release_latest.generic_all.deb
+        $ sudo apt update
+        $ sudo percona-release setup ps80
```

Comment:
`ps80` will not include innovative releases.
The last time we discussed this the name should be `ps-8x-innovative` but this needs to be clarified with Evgeniy's team.

Reply:
Right - all of this is unknown at this time. Can fix after the release, as needed.